common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
+                            "You agree to send a closing email at the conclusion of the course to communicate final course details."
                         ),
                         "is_checked": False,
-                        "action_url": "SettingsDetails",
-                        "action_text": _("Edit Course Schedule &amp; Details"),
+                        "action_url": "CourseOutline",
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Add Staff Bios"),
+                        "short_description": _("Instruct learners to introduce themselves"),
                         "long_description": _(
-                            "Showing prospective students who their instructor will be is helpful. "
-                            "Include staff bios on the course About page."
+                            "Encourage learners to use the discussion forums to introduce themselves."
                         ),
                         "is_checked": False,
-                        "action_url": "SettingsDetails",
-                        "action_text": _("Edit Course Schedule &amp; Details"),
+                        "action_url": "",
+                        "action_text": "",
                         "action_external": False,
                     },
+                ],
+            },
+             {
+                "short_description": _("Re-runs, reusability, and licensing"),
+                "items": [
                     {
-                        "short_description": _("Add Course FAQs"),
-                        "long_description": _("Include a short list of frequently asked questions about your course."),
+                        "short_description": _("Clear IP issues"),
+                        "long_description": _(
+                            "All IP issues are cleared for the course and clearly documented."
+                        ),
                         "is_checked": False,
-                        "action_url": "SettingsDetails",
-                        "action_text": _("Edit Course Schedule &amp; Details"),
+                        "action_url": "",
+                        "action_text": "",
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Add Course Prerequisites"),
+                        "short_description": _("Complete Course Details worksheet"),
                         "long_description": _(
-                            "Let students know what knowledge and/or skills they should have before "
-                            "they enroll in your course."
+                            "Complete the Course Details worksheet which documents any IP issues or uses of third-party content or tools."
```

Comment:
@lamagnifica it will be on the portal, for now just provided by PMs: https://docs.google.com/document/d/1w43J894YmVzDjqfHIMtbTmabxxiejnbE4j-XPSnU3LE/edit?usp=sharing